harmonica/forward/prism.py

```diff
@@ -186,7 +208,7 @@ def _check_prisms(prisms):
         raise ValueError(err_msg)
 
 
-def jit_prism_gravity(coordinates, prisms, density, kernel, out):
+def jit_prism_gravity(coordinates, prisms, density, kernel, out, progress_proxy):
```

Comment:
Is it possible to make `progress_proxy` an optional parameter with default to `None`? We are not currently obliged to keep backwards compatibility, and specially not with private functions, but since it's not required to perform the computation, sounds better if we keep it optional. If Numba complains about it, we can go back, no problem.

```suggestion
def jit_prism_gravity(coordinates, prisms, density, kernel, out, progress_proxy=None):
```

Reply:
Yes that seems to still work so I'll add your suggestion in.